Dockerfile

```diff
@@ -1,5 +1,12 @@
-# select build image
-FROM rust:1.41 as build
+ARG BUILD_IMAGE
+
+FROM ${BUILD_IMAGE} as build
+
+ARG REPO
+ARG VCS_REF
+
+LABEL com.logdna.stage="${REPO}-build"
+LABEL com.logdna.tag="${VCS_REF}"
```

Comment:
This label allows the cleaning to be more specific so we can safely have concurrent jenkins builds